src/tests/utils/ApiTestSteps.ts

```diff
@@ -243,3 +243,27 @@ export async function validateTxMAEventData(keyList: any): Promise<any> {
 			});
 	}
 }
+
+export function validateJwtToken(jwtToken: any): void {
```

Comment:
You're using await validateJwtToken() in your test steps but the function you're calling is not asynchronous.